TLM/TLM/Resources/lang_fr.txt

```diff
@@ -175,60 +175,60 @@ default par défaut
 flow_ratio ratio de mouvement
 wait_ratio ratio d'attente
 After_min._time_has_elapsed_switch_to_next_step_if Après le temps minimum écoulé, passez à l'étape suivante si
-Adaptive_step_switching Commutation d'étape adaptive
+Adaptive_step_switching Commutation d'étape adaptative
 Dynamic_lane_section Sélection de voie dynamique
 Percentage_of_vehicles_performing_dynamic_lane_section Pourcentage de véhicules effectuant une sélection de voie dynamique
 Vehicle_restrictions_aggression Restrictions agressives des véhicules
 Strict Stricte
-Show_path-find_stats Afficher le chemin d'acès des stats
+Show_path-find_stats Afficher le chemin d'accès des stats
 Remove_this_vehicle Retirer ce véhicule
-Vehicles_follow_priority_rules_at_junctions_with_timed_traffic_lights Les véhicules suivent les règles de priorité aux carrefours avec des feux de circulations temporiés
-Enable_tutorial_messages Activier les messages du didactitiel
-TMPE_TUTORIAL_HEAD_MainMenu Gestionnaire de traffic : Édition Président
-TMPE_TUTORIAL_BODY_MainMenu Bienvenue à TM:PE !\n\nManuel d'utilisateur : http://www.viathinksoft.de/tmpe
+Vehicles_follow_priority_rules_at_junctions_with_timed_traffic_lights Les véhicules suivent les règles de priorité aux carrefours avec des feux de circulations chronométrés
```

Comment:
Should it be `suivant` instead of `suivent` (`a` not `e`)?

Reply:
Translated from `Vehicles follow priority rules at junctions with timed traffic lights`

If an `a` is used, it would mean "Vehicles _which are following_ prority rules [...]"